src/main/java/com/avairebot/commands/music/RemoveSongFromQueueCommand.java

```diff
@@ -71,7 +71,7 @@ public String getDescription() {
 
     @Override
     public List<String> getTriggers() {
-        return Arrays.asList("removesong", "songremove");
+        return Arrays.asList("removesong", "songremove", "removesongs");
```

Comment:
Why is this here? Doesn't seem related to the PR.

Reply:
I meant to add this into my previous PR when I added the ability to remove multiple songs but I forgot to do that, so I decided to throw that into this PR while I was at it. I can remove it if you want me to if you don't find this necessary.

Reply:
Although it doesn't really fit the PR, it should be fine to just leave it, unless you wanna make another PR for a single line code change :p

Reply:
It seems a bit of a waste of time to create a PR for a single added word, so I'll leave it here while I'm at it.